From 960c42193acdab214217dd8a472f7f28d7ca5153 Mon Sep 17 00:00:00 2001
From: Marian Beermann <public@enkore.de>
Date: Sun, 13 Nov 2016 11:19:58 +0100
Subject: [PATCH] fix tox build for environment-python != containing-python in
 yet-another instance

this instance: the repository worktree is *not* named borg.

Cherry pick of 4f1157c into 1.0-maint due to f3efcdb

TODO removed since we already did that after 1.0-maint, but 1.0-maint
will never receive the change.
---
 conftest.py | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/conftest.py b/conftest.py
index d80aed4..d542366 100644
--- a/conftest.py
+++ b/conftest.py
@@ -1,4 +1,5 @@
 import os
+import os.path
 import sys
 
 import pytest
@@ -10,12 +11,10 @@
 # The workaround is to remove entries pointing there from the path and check whether "borg"
 # is still importable. If it is not, then it has not been installed in the environment
 # and the entries are put back.
-#
-# TODO: After moving the package to src/: remove this.
 
 original_path = list(sys.path)
 for entry in original_path:
-    if entry == '' or entry.endswith('/borg'):
+    if entry == '' or entry == os.path.dirname(__file__):
         sys.path.remove(entry)
 
 try:
